pkg/server/server.go

```diff
@@ -750,6 +777,30 @@ func (s *Server) indexHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) cspReportsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Note - it is possible for Console frontend to send CSP violation reports directly to the appropriate service on the cluster.

If we decide to do that, we don't need this handler.